CHANGELOG.md

```diff
@@ -52,6 +53,11 @@ Changed
 - Upgraded dependency to use boto3>=1.13.5. (#57161)
 - Changed to consistent file location handling across APIs for Juniper network devices. (#57399)
 - Use Python's hashlib (sha256) instead of shelling out (SipHash24) to generate server_id. (#57415)
+- Some Debian network (debian_ip) refactoring (#49462)
+- Port #53537 to master (#56835)
```

Comment:
This is already in here - I'm suspecting the other ones will be as well - https://github.com/saltstack/salt/pull/53537 fixes issue https://github.com/saltstack/salt/issues/4438